websites/F/Font Awesome/presence.ts

```diff
@@ -0,0 +1,63 @@
+const pPresence = new Presence({
+  clientId: "820023496934817804"
+}),
+  pBrowsing = Math.floor(Date.now() / 1000);
+
+pPresence.on("UpdateData", async () => {
+  const pData: PresenceData = {
+    largeImageKey: "icon2"
+  },
+    pPage = window.location.pathname;
+
+  pData.startTimestamp = pBrowsing;
```

Comment:
oh no wait, sorry for that but could you merge it together 🤣  to pData object.

Reply:
Idk what you mean with that?

Reply:
Please tell me what you mean with that @ririxidev

Reply:
```suggestion
    largeImageKey: "icon2",
    startTimestamp: pBrowsing
  },
    pPage = window.location.pathname;
```

Reply:
SlowLife already answered.

Reply:
> SlowLife already answered.

fixed